index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Project1</title>
+		<script src="https://kit.fontawesome.com/62fc5eef3c.js" crossorigin="anonymous"></script>
+	</head>
+	<body>
+		<!--Navbar -->
+		<header>
+			<nav>
```

Comment:
what is the purpose of having both header and nav tags here? would it be possible to reduce it to only 1 tag?